pkg/sql/lease.go

```diff
@@ -46,9 +47,16 @@ import (
 
 var (
 	// LeaseDuration is the mean duration a lease will be acquired for. The
-	// actual duration is jittered in the range
-	// [0.75,1.25]*LeaseDuration. Exported for testing purposes only.
+	// actual duration is jittered using LeaseJitterFraction. Jittering is done to
+	// prevent multiple leases from being renewed simultaneously if they were all
+	// first acquired simultaneously. Exported for testing purposes only.
 	LeaseDuration = 5 * time.Minute
+	// LeaseJitterFraction is the factor that we use to randomly jitter the lease
+	// duration when acquiring a new lease and the lease renewal timeout. The
+	// range of the actual lease duration will be
+	// [(1-LeaseJitterFraction) * LeaseDuration, (1+LeaseJitterFraction) * LeaseDuration]
+	// Exported for testing purposes only.
+	LeaseJitterFraction = 0.25
```

Comment:
I'd add this in the same PR that uses it. It makes sense to not make this change here.